src/dashboard.Dockerfile

```diff
@@ -34,8 +34,6 @@ ADD dashboard/src/requirements/ /src/dashboard/src/requirements/
 RUN pip install -r /src/dashboard/src/requirements/production.txt -r /src/dashboard/src/requirements/dev.txt
 ADD dashboard/ /src/dashboard/
 
-ADD archivematicaCommon/etc/dbsettings /etc/archivematica/archivematicaCommon/dbsettings
-
 RUN set -ex \
 	&& groupadd -r archivematica \
 	&& useradd -r -g archivematica archivematica
```

Comment:
I found that the `ARCHIVEMATICA_DASHBOARD_DASHBOARD_GEARMAN_SERVER` is also necessary in the `collectstatic` command in this file in order for the build to run, as the settings file tries to reference it from the config.